src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/AddressLogEntry.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Sylius Sp. z o.o.
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<resources
+    xmlns="https://api-platform.com/schema/metadata/resources-3.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="https://api-platform.com/schema/metadata/resources-3.0 https://api-platform.com/schema/metadata/resources-3.0.xsd"
+>
+    <resource class="Sylius\Component\Addressing\Model\AddressLogEntry" shortName="AddressLogEntry">
+        <operations>
+            <operation
+                class="ApiPlatform\Metadata\Get"
+                controller="ApiPlatform\Action\NotFoundAction"
+                read="false"
+                output="false"
+                routePrefix="admin"
+            />
```

Comment:
I wonder if it doesn't make sense to also retrieve individually 🤔  but we can think about that in the future, it wasn't possible before either